lib/dev/ui/spinner.rb

```diff
@@ -40,6 +40,7 @@ def initialize(title, &block)
               end
             end
 
+            @force_full_render = false
```

Comment:
This is a good point -- it would make more sense to render immediately, off the timer -- but then we'd have to mutex the render function and it would make the code quite a bit more complex.

I think this is the simplest solution, but we should realize that it comes at the price of changes taking a few dozen ms to appear.